http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterToolMonitorSpec.scala

```diff
@@ -42,34 +44,65 @@ class ClusterToolMonitorSpec
     super.afterAll()
   }
 
-  val welderEnabledCluster = makeCluster(1).copy(status = ClusterStatus.Running, welderEnabled = true)
-  val welderDisabledCluster = makeCluster(2).copy(status = ClusterStatus.Running, welderEnabled = false)
-  val notRunningCluster = makeCluster(3).copy(status = ClusterStatus.Deleted, welderEnabled = true)
-
-  it should "report both services are up normally" in isolatedDbTest {
+  val welderEnabledCluster = makeCluster(1).copy(status = ClusterStatus.Running,
+                                                 welderEnabled = true,
+                                                 clusterImages = Set(jupyterImage, welderImage))
+  val welderDisabledCluster =
+    makeCluster(2).copy(status = ClusterStatus.Running, welderEnabled = false, clusterImages = Set(jupyterImage))
+  val notRunningCluster = makeCluster(3).copy(status = ClusterStatus.Deleted,
+                                              welderEnabled = true,
+                                              clusterImages = Set(jupyterImage, welderImage))
+  val rstudioCluster = makeCluster(4).copy(status = ClusterStatus.Running,
+                                           welderEnabled = true,
+                                           clusterImages = Set(rstudioImage, welderImage))
+
+  it should "report all services are up normally" in isolatedDbTest {
     welderEnabledCluster.save()
+    welderDisabledCluster.save()
+    notRunningCluster.save()
+    rstudioCluster.save()
 
     withServiceActor() { (_, mockNewRelic) =>
       Thread.sleep(clusterToolConfig.pollPeriod.toMillis * 3)
-      verify(mockNewRelic, never()).incrementCounterFuture(ArgumentMatchers.startsWith("JupyterDown"),
-                                                           ArgumentMatchers.anyInt())(ArgumentMatchers.any())
-      verify(mockNewRelic, never()).incrementCounterFuture(ArgumentMatchers.startsWith("WelderDown"),
```

Comment:
These tests had some problems… for one it was verifying `incrementCounterFuture`, which is not actually called by `ClusterToolMonitor`